index.html

```diff
@@ -5081,9 +5031,60 @@ <h2 id="AudioWorkletNode">
             connected with other built-in <a><code>AudioNode</code></a>s to
             form an audio graph.
           </p>
-          <dl title=
-          "[Constructor(BaseAudioContext context, DOMString name, optional AudioWorkletNodeOptions options)] interface AudioWorkletNode : AudioNode"
-          class="idl">
+          <dl title="interface AudioWorkletNode : AudioNode" class="idl">
+            <dt>
+              Constructor(BaseAudioContext context, DOMString name, optional
+              AudioWorkletNodeOptions options)
+            </dt>
+            <dd>
+              <p>
```

Comment:
Not really sure this is the best place to describe the construction.  I expect these to be relatively short descriptions of the item.